src/main/resources/static/api.yaml

```diff
@@ -1335,6 +1335,54 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+  /patients/profile/logintype:
```

Comment:
if we are changing the Method GET to POST, then we should call the end point as /patients/profile/enquire-login or so.